tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -38,6 +38,10 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Ensure that there is enough free memory for virtual network test
+    my $MEM = $self->get_free_mem();
+    die "FREE MEM=$MEM. The SUT needs at least 20G of RAM" if $MEM < 20000;
```

Comment:
I think you'd better user record_info("Detect memrory"); assert_script_run("test $mem >= 20G") which serves both assertion and keep run history record instead of useing die directly.

Reply:
You should append unit to mem size in your message output.

Reply:
Cool, had add `record_info()`

Reply:
DONE